server/index.js

```diff
@@ -75,22 +77,24 @@ const caAndDmNotificationSender = createCaAndDmNotificationSender(
   config
 )
 
+const eventPublisher = new EventPublisher(audit, appInsights)
```

Comment:
Good idea.

Reply:
Yes, I could have done this - was just being a bit more specific. I'll have a look in a follow up